src/tacacs/pam/0010-handle-bad-password-set-by-sshd.patch

```diff
@@ -1,20 +1,20 @@
-From 4ef102dfa137c2c6e3dd63c2d16f2966b6f5add8 Mon Sep 17 00:00:00 2001
```

Comment:
The reason for rename and change this file is:
  This file been backport from master to 202012 by PR #10032
  Because in master branch, this file been add after PR #8659,  so when backport some code changed because #8659 not backport to 202012.
  So, when backport PR #8659 to 202012 by current PR, we need change back this file. what I do is copy the file from master branch.